docs/community-libraries-and-navigators.md

```diff
@@ -59,3 +59,20 @@ Provides simple HOCs that map react-navigation props to your screen components d
 #### Links
 
 [github.com/vonovak/react-navigation-props-mapper](https://github.com/vonovak/react-navigation-props-mapper)
+
+## react-native-header-scroll-view
+
+This component addresses the most requested feature to React Navigation – [iOS large header and grow/shrink on scroll](https://react-navigation.canny.io/feature-requests/p/ios-11-large-header-and-growshrink-on-scroll), made by [@jonsamp](https://github.com/jonsamp). It is different than the actual iOS large header behavior, but does the header title shrinking well. It can be used with React Navigation by using the following code in your component:
+
+```
+  static navigationOptions = {
+    headerMode: "none",
```

Comment:
`headerMode` is an option in `createStackNavigator`'s config, not in `navigationOptions`. I'd document both, `headerMode: 'none'` to disable globally, and `header: null` to disable header for one screen.

Reply:
done!